apps/merkle_patricia_tree/mix.exs

```diff
@@ -51,6 +51,7 @@ defmodule MerklePatriciaTree.Mixfile do
       {:ex_rlp, "~> 0.3.0"},
       {:exth_crypto, in_umbrella: true},
       {:exleveldb, "~> 0.13.1"},
```

Comment:
Can we remove the dependency on `exleveldb` now? I can't think of a reason anyone would want to use `leveldb` instead of `rocksdb` now.

If we do want to keep it we could probably make it [optional](https://hexdocs.pm/mix/Mix.Tasks.Deps.html#module-dependency-definition-options).

Reply:
Yea, good point